bin/pftt2-release.bat

```diff
@@ -0,0 +1,34 @@
+@ECHO OFF
+
+REM Check if build directory exists
+cd ../
+if exist build (
+	REM Create pftt-release directory in main folder
+	mkdir pftt-release
+
+	REM Create pftt2.jar and place in lib folder
+	cd build
+	jar cf pftt2.jar ./com/ ./org/columba ./org/kxml2 ./org/incava
+	MOVE ./pftt2.jar ../lib
```

Comment:
It would be better to move the jar right into the package. Alternatively, it could be deleted after the packaging. It should not stay laying around, as that might cause quite some WTF moments when you develop but the script picks up the packaged jar :)

Thanks.

Reply:
Updated to move the jar file directly into the package instead. Please check